parsing/parsetree.mli

```diff
@@ -24,21 +24,20 @@ open Asttypes
 
 type constant =
     Pconst_integer of string * char option
-  (* 3 3l 3L 3n
+  (** Represents integer constants such as [3] [3l] [3L] [3n].
```

Comment:
I am not convinced by this format: `represents` is repeated often enough to become noise.
Maybe a more minimal change would be:

 ```ocaml
 [3], [3l], [3L], [3n] : integer constants
 ```

Reply:
I removed noise occurrences of "Represents" but kept a sentence instead of a `:` which I found in some cases confusing. Let me know if that is sufficiently compact.